cameraview/src/main/java/com/otaliastudios/cameraview/CameraView.java

```diff
@@ -1308,6 +1308,7 @@ public boolean getPlaySounds() {
         void dispatchOnFocusEnd(@Nullable Gesture trigger, boolean success, PointF where);
         void dispatchOnZoomChanged(final float newValue, final PointF[] fingers);
         void dispatchOnExposureCorrectionChanged(float newValue, float[] bounds, PointF[] fingers);
+        void onError(String message, Exception cause);
```

Comment:
This is not fundamental, but I think it's better if this would accept directly a `CameraException`, so who first gets the error must instantiate a CameraException and pass it here.

This would be useful if one day we want to add more functionality into `CameraException` (for example an integer describing the kind of error)